CalibTracker/SiStripLorentzAngle/interface/TIBorTOBparameters.h

```diff
@@ -0,0 +1,21 @@
+#ifndef TIBORTOBPARAMETERS_H
+#define TIBORTOBPARAMETERS_H
+
+#include "DataFormats/SiStripDetId/interface/SiStripDetId.h"
+
+// WARNING: This header file is created on the assumption
+// that the subDetector Id (TIB or TOB) is already known.
+// It is used inside CalibTracker/SiStripLorentzAngle only.
```

Comment:
maybe "see `git show CMSSW_9_2_3_patch2:DataFormats/SiStripDetId/interface/TIBDetId.h` and `git show CMSSW_9_2_3_patch2:DataFormats/SiStripDetId/interface/TOBDetId.h`", or is that too detailed?